### PR TITLE
feat: sync with origin v2.7.0

### PR DESCRIPTION
**Pull request by fauzanlinnas**
> This project is a forked version of the [Sendbird UI Kit](https://github.com/sendbird/sendbird-uikit-react) to match with the Ruangguru's internal requirements. Therefore, it is not yet set up to accept pull requests from external contributors. But you can always freely create a forked version from this repository to match your own requirements.

### Demo
https://user-images.githubusercontent.com/26358930/165702234-d87f37a9-5a9c-4479-993f-2bb637b82363.mp4

### CODE REVIEW

**Comment by fauzanlinnas** [comment]
oiya blm cek snapshotnya

**Comment by suliskh** [comment]
> oiya blm cek snapshotnya

There're failing tests that block the merging, Mas @fauzanlinnas. Pls fix that yah :sungkem:

**Comment by fauzanlinnas** [comment]
> > oiya blm cek snapshotnya
> 
> There're failing tests that block the merging, Mas @fauzanlinnas. Pls fix that yah :sungkem:

ku bingung nih kenapa
kalo jalanin `npm run test` di local berhasil semua, tapi di githubnya ngga @suliskh 

di local
image


di github
image

File: package.json [hunk 1]
```diff
@@ -108,7 +107,9 @@
     "np": "^7.5.0",
     "plop": "^2.5.3",
     "postcss": "^8.3.5",
-    "react-test-renderer": "^16.8.6",
+    "react": "^17.0.2",
```
Comment (suliskh) [inline package.json]: Dicoba `react-test-renderer`, `react`, `react-dom`-nya tetep versi yg 16 yaa. Karena di Ruangkelas juga masih pakai 16

**Review by suliskh — CHANGES_REQUESTED** [review]

File: src/rogu/ui/FileMessageItemBody/__tests__/FileMessageItemBody.spec.js [hunk 2]
```diff
@@ -5,6 +5,16 @@ import FileMessageItemBody from '../index';
 
 import { FILE_MESSAGE_DOC } from '../../../../../__mocks__/messagesMock';
 
+const createMockMessage = (process) => {
```
Comment (suliskh) [inline src/rogu/ui/FileMessageItemBody/__tests__/FileMessageItemBody.spec.js]: Is it expected changes?

File: rollup.config.js [hunk 3]
```diff
@@ -35,12 +35,12 @@ module.exports = ({
   },
   output: [
     {
-      dir: 'release/dist/cjs',
```
Comment (suliskh) [inline rollup.config.js]: I think it is better to not change the build path. Make sure there's no regression by installing it in Ruangkelas

File: src/rogu/ui/FileMessageItemBody/__tests__/FileMessageItemBody.spec.js (hunk 2 above)
Comment (fauzanlinnas) [reply src/rogu/ui/FileMessageItemBody/__tests__/FileMessageItemBody.spec.js]: deleted karena ngga kepake

File: rollup.config.js (hunk 3 above)
Comment (fauzanlinnas) [reply rollup.config.js]: all path revert to ruangkelas config

**Comment by suliskh** [comment]
Letsgoo 🚀 

Sebelum rilis, minta tolong dicoba build dan pasang di Ruangkelas yah mas :sungkem:

**Review by suliskh — APPROVED** [review]

**Pull request merged**